2017-01-04 16:29:26,451 INFO 1395 [dd.dogapi] dog_stats_api.py:66 - Initializing dog api to use statsd: localhost, 8125
2017-01-04 16:29:26,694 WARNING 1395 [py.warnings] base.py:260 - /opt/bitnami/apps/edx/edx-platform/venv/lib/python2.7/site-packages/django/core/management/base.py:260: RemovedInDjango19Warning: "requires_model_validation" is deprecated in favor of "requires_system_checks".
  RemovedInDjango19Warning)

System check identified some issues:

WARNINGS:
wiki.ArticleRevision.ip_address: (fields.W900) IPAddressField has been deprecated. Support for it (except in historical migrations) will be removed in Django 1.9.
	HINT: Use GenericIPAddressField instead.
2017-01-04 16:29:34,325 WARNING 1395 [py.warnings] worker.py:161 - /opt/bitnami/apps/edx/edx-platform/venv/lib/python2.7/site-packages/celery/apps/worker.py:161: CDeprecationWarning: 
Starting from version 3.2 Celery will refuse to accept pickle by default.

The pickle serializer is a security concern as it may give attackers
the ability to execute any command.  It's important to secure
your broker from unauthorized access when using pickle, so we think
that enabling pickle should require a deliberate action and not be
the default choice.

If you depend on pickle then you should set a setting to disable this
warning and to be sure that everything will continue working
when you upgrade to Celery 3.2::

    CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']

You must only enable the serializers that you will actually use.


  warnings.warn(CDeprecationWarning(W_PICKLE_DEPRECATED))

2017-01-04 16:29:34,493 ERROR 1395 [celery.worker.consumer] consumer.py:364 - consumer: Cannot connect to amqp://bn_edx:**@127.0.0.1:5672//: [Errno 111] Connection refused.
Trying again in 2.00 seconds...

2017-01-04 16:29:36,504 ERROR 1395 [celery.worker.consumer] consumer.py:364 - consumer: Cannot connect to amqp://bn_edx:**@127.0.0.1:5672//: [Errno 111] Connection refused.
Trying again in 4.00 seconds...

2017-01-04 16:29:40,537 INFO 1395 [celery.worker.consumer] consumer.py:479 - Connected to amqp://bn_edx:**@127.0.0.1:5672//
2017-01-04 16:29:40,571 INFO 1395 [celery.worker.consumer] consumer.py:567 - mingle: searching for neighbors
2017-01-04 16:29:41,598 INFO 1395 [celery.worker.consumer] consumer.py:573 - mingle: sync with 5 nodes
2017-01-04 16:29:41,599 INFO 1395 [celery.worker.consumer] consumer.py:583 - mingle: sync complete
2017-01-04 16:29:41,629 WARNING 1395 [celery.redirected] log.py:282 - celery@edx.cms.core.high.bitnami-edx-dm-dd2b ready.
